SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -108,20 +106,6 @@ - (void)sdl_backgroundTaskEnd {
     self.backgroundTaskId = UIBackgroundTaskInvalid;
 }
 
-#pragma mark session setup getter/setter
-
-- (void)setSessionSetupInProgress:(BOOL)inProgress{
-    _sessionSetupInProgress = inProgress;
-    if (!inProgress){
-        // End the background task here to catch all cases
-        [self sdl_backgroundTaskEnd];
```

Comment:
Yes it was, but I wasn't fully clear on what you were asking me to remove. Can you clarify?